types.cc

```diff
@@ -3455,6 +3455,16 @@ static std::string unescape(sstring_view s) {
     return result;
 }
 
+// Replace ":" with "\:" and "@" with "\@".
+static std::string escape(sstring_view s) {
+    static thread_local std::regex colon_re(":");
+    static thread_local std::regex at_re("@");
+    std::string result(s);
```

Comment:
No need for thread_local since it's not mutable (can add const to stress that).

Reply:
Done.